src/electronApp/angularApp/components/connect-form/connect-form.component.ts

```diff
@@ -1,35 +1,56 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
-import { PrinterService } from '../../services/printerService';
-import { ErrorLogger } from '../../../core/errorLogger';
-import { FormControl, FormGroupDirective, NgForm, Validators} from '@angular/forms';
-import { ErrorStateMatcher} from '@angular/material/core';
-import { Router, ActivatedRoute } from '@angular/router';
+import {
```

Comment:
Can you update this, please, with your new formatter settings so there are fewer unrelated changes :)